packages/pigment-css-react/src/utils/generateCss.ts

```diff
@@ -2,34 +2,8 @@ import { serializeStyles } from '@emotion/serialize';
 import { Theme } from './extendTheme';
 
 export function generateTokenCss(theme: Theme) {
```

Comment:
Can you also add tests for these files now that the testing is set-up.

Reply:
Done in https://github.com/mui/material-ui/pull/41446/commits/0bba432b42b31769be3dda34879f94ecb958b172